cms/urls.py

```diff
@@ -72,6 +72,7 @@
         r'^course_info_update/{}/(?P<provided_id>\d+)?$'.format(settings.COURSE_KEY_PATTERN),
         'course_info_update_handler'
     ),
+    url(r'^home/$', 'course_listing', name='home'),
```

Comment:
@cahrens Yes, it's handling the redirect in https://github.com/open-craft/edx-platform/blob/content_libraries/11-UI-updates/cms/djangoapps/contentstore/views/course.py#L235